internal/ais/workflow.go

```diff
@@ -188,6 +188,20 @@ func (w *Workflow) SessionHandler(ctx temporalsdk_workflow.Context, aipUUID, aip
 		return "", e
 	}
 
+	var combineMDResult CombineMDActivityResult
+	e = temporalsdk_workflow.ExecuteActivity(
+		withFilesystemActivityOpts(ctx),
+		CombineMDActivityName,
+		&CombineMDActivityParams{
+			AreldaRelPath: metadataRelPath,
```

Comment:
The absolute path is the destination used in the previous activity params.

Reply:
Again, I found the paths and naming convention used in the workflow very unclear.

Reply:
I agree, it would be nice to add a state struct where we can add some comments about the variables. An improvement for another PR.